app/core/config.py

```
@@ -0,0 +1,293 @@
+"""
+Backend configuration module for Mobius Context Platform.
+
+This module provides type-safe configuration management using Pydantic Settings,
+supporting environment variables, .env files, and environment-specific configurations.
+All environment variables are prefixed with MOBIUS_ for namespace isolation.
+"""
+
+from typing import List
```

Ruff (0.11.9)

`typing.List` is deprecated, use `list` instead

(UP035)